packages/server/src/fhir/repo.ts

```diff
@@ -1898,6 +1904,36 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
     }
   }
 
+  private sameReferences(a: Reference[] | undefined, b: Reference[] | undefined): boolean {
```

Comment:
We might be able to do something closer to the original code, like:

`a.difference(b).size === 0 && b.difference(a).size === 0`

Reply:
True, I was just trying to avoid allocating the arrays from `.difference()`

Reply:
Fair,

If we can at least get a test case that represents what we are trying to log then I think we can get enough coverage for this

I guess scenarios could be:
* Create non-Patient resource with 1 account
* Have a project admin remove that account

* Create non-Patient resource with 1 account
* Have a project admin add another account

Something like that?